tasks/test-project/codemods/aboutPage.js

```diff
@@ -9,14 +9,24 @@ export default (file, api) => {
   const j = api.jscodeshift
   const root = j(file.source)
 
+  // Remove the `{ Link, routes }` imports that are generated and unused
```

Comment:
It's comments like these that makes me really appreciate our codebase. It makes it much more approachable than a lot of other open source code I've looked at